subsys/net/lib/nrf_provisioning/Kconfig

```diff
@@ -38,6 +38,12 @@ config NRF_PROVISIONING_WITH_CERT
 	help
 	  Includes the root certificate used by the server side and provisions it if needed.
 
+config NRF_PROVISIONING_SAVE_CMD_ID
+	bool "Save the latest command id to storage"
+	default n
```

Comment:
Should this be y, as we were already doing it? I tried to look what was being done with the stored value and to me it seemed that it was only used for informational purposes at the moment. If there is no real usage, then the default value does not matter.

Reply:
Btw, defaulting to n is redundant as it is by default n.

Reply:
Before this change the command id was saved to storage and loaded when the client starts. After some internal discussion, we decided to change this to optional and not to enable by default. This allows the customer to re-send the full provisioning configuration.